cmd/promql-langserver/promql-langserver.go

```diff
@@ -23,6 +23,21 @@ import (
 
 	"github.com/prometheus-community/promql-langserver/langserver"
 	"github.com/prometheus-community/promql-langserver/rest"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
```

Comment:
rather than define these http-specific metrics here, can we do this in the rest package inside of the `CreateHandler` func? That would help keep the main package cleaner

Reply:
In general all theses metrics should be defined in the rest package, since they should be usable from anywhere the rest package is imported.

Reply:
okay